Tests/AWSSDKSwiftCoreTests/PaginateTests.swift

```diff
@@ -26,13 +26,15 @@ class PaginateTests: XCTestCase {
     var eventLoopGroup: EventLoopGroup!
     var httpClient: AWSHTTPClient!
     var client: AWSClient!
+    var config: AWSServiceConfig!
```

Comment:
no need to wait until the `setUp` function to create this. Although this is fine as well. leave it if you want, as you like.

Reply:
my intention was to put the setup code all in one place together. but you are right, we can assign it directly at the declaration, maybe also make it a let. I'll see

Reply:
the service config uses the address of the server which is initialized in `setup`. so I would just leave the creation of config in the setup as well. other options would be to make it a lazy var but I think the way it is now is more readable.

Reply:
fair point